Makefile

```diff
@@ -35,5 +35,20 @@ tests_common_gpu:
 	python -m pytest tests/test_decoder_models.py $(if $(IS_GITHUB_CI),--report-log "common_decoder.log",)
 	python -m pytest tests/test_encoder_decoder_models.py $(if $(IS_GITHUB_CI),--report-log "common_encoder_decoder.log",)
 
+tests_examples_multi_gpu_bnb:
+	python -m pytest -m "multi_gpu_tests and bitsandbytes" tests/test_gpu_examples.py $(if $(IS_GITHUB_CI),--report-log "multi_gpu_examples.log",)
```

Comment:
Let's hope we always mark the bnb tests, it's easy to forget and I'm not sure if we would see failures necessarily.

Reply:
It should have been done here: https://github.com/huggingface/peft/pull/1242